### DISCUSSION TITLE
Safely access js attributes

### DISCUSSION THREAD

**Discussion by thewh1teagle** [brython-dev/brython#2455] [Q&A]
I'm trying to access js values in global objects on window, safely
```python
from browser import window
window["SOME_GLOBAL"].get('get_key_safely' None)
```
When I tried that, I got the error
`AttributeError: 'Javascript Object' object has no attribute 'get'`

How can we use objects on global window and safely access it like with `dict` `get` method? including on the `window` itself.

**Comment by PierreQuentel** [comment]
In Brython code, use Python's `getattr()` function : 

```python
from browser import window

print(getattr(window.SOME_GLOBAL, 'a', None))
```
